cdap-ui/app/cdap/components/PipelineList/DeployedPipelineView/NextRun/index.tsx

```diff
@@ -62,23 +58,26 @@ export default class NextRun extends React.PureComponent<IProps, IState> {
   }
 
   private getNextRun() {
-    const namespace = getCurrentNamespace();
-    const pipeline = this.props.pipeline;
-
-    const params = {
-      ...DataPipelineProgram,
-      namespace,
-      appId: pipeline.name,
-    };
-
-    MyPipelineApi.getNextRun(params).subscribe((res) => {
-      this.setState({
-        loading: false,
-        nextRun: res.length ? res[0].time : null,
-      });
+    const latestSchedule = this.getLatestSchedule(this.props.pipeline);
+    const nextRuntimes = objectQuery(latestSchedule, 'nextRuntimes');
```

Comment:
you're using the object query wrong. 
`objectQuery(getProgram(this.props.pipeline), 'schedules', 0, 'nextRuntimes')`

Please fix the other occurrence of `objectQuery` on this PR

Reply:
Could you please check if I am using the function correctly now?

Reply:
i don't see the update?